docs/commands/alias.md

```diff
@@ -0,0 +1,62 @@
+# alias
+This command allows you to define shortcuts for common commands, equivalent to the same bash command.
```

Comment:
May want to switch "equivalent" to "similar", since we don't append alias arguments to the end of the alias body.

That being said, I'm feeling like we shouldn't reference things from other shells in these docs, and leave that for our book and other documentation.

Reply:
I removed the bash reference. Added "other" in front of "common commands", since the sentence already starts with "This command".